.snippets/code/develop/toolkit/api-libraries/polkadot-js-sdk/send-txs.js

```diff
@@ -0,0 +1,10 @@
+// Assuming you have an `alice` keypair from the Keyring
+const RECIPIENT = 'RECIPIENT_ADDRESS';
+const AMOUNT = 12345; // Amount in the smallest unit (e.g., Planck for DOT)
+
+// Sign and send a transfer
+const txHash = await api.tx.balances
+  .transfer(RECIPIENT, AMOUNT)
+  .signAndSend(alice);
```

Comment:
Nit picking here, but I think it would be cool to indent as follows:

```suggestion
const txHash = await api.tx
  .balances
  .transfer(RECIPIENT, AMOUNT)
  .signAndSend(alice);
```

So it's even clearer the pallet and method that the library is calling